runtime/doc/options.txt

```diff
@@ -4320,20 +4319,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	reset this option. |-u| |--noplugin|
 
 						*'macatsui'* *'nomacatsui'*
-'macatsui'		boolean	(default on)
-			global
-			{only available in Mac GUI version}
-	This is a workaround for when drawing doesn't work properly.  When set
-	and compiled with multi-byte support ATSUI text drawing is used.  When
-	not set ATSUI text drawing is not used.  Switch this option off when
-	you experience drawing problems.  In a future version the problems may
-	be solved and this option becomes obsolete.  Therefore use this method
-	to unset it: >
-		if exists('&macatsui')
-		   set nomacatsui
-		endif
-<	Another option to check if you have drawing problems is
-	'termencoding'.
+'macatsui'		Removed. |vim-differences| {Nvim}
```

Comment:
IIRC it's been done for at least a few months, even before `vim-diff.txt`, e.g., 2bf085049718a163ddc71b8c02989158ca595596 and 497a688ebc115c6535360f08464309ede9fba0cf.

> Why would we want to keep a stub of this legacy option around?

So users know that we haven't accidentally removed it (which we've done with some commands), and because a stub saying `Removed.` is arguably more helpful than `Sorry, no help  for salkdjsakldsa`. They take up very little room, so I'd rather we keep them for the practical benefit, at least for the time being.

What do you think?